spec.html

```diff
@@ -7851,13 +7851,15 @@ <h1>[[DefineOwnProperty]] ( _P_, _Desc_ )</h1>
       </emu-clause>
 
       <emu-clause id="sec-arraycreate" aoid="ArrayCreate">
-        <h1>ArrayCreate ( _length_ [ , _proto_ ] )</h1>
+        <h1>ArrayCreate ( _length_ [ , _proto_ [ , _internalSlotsList_ ] ] )</h1>
         <p>The abstract operation ArrayCreate with argument _length_ (either 0 or a positive integer) and optional argument _proto_ is used to specify the creation of new Array exotic objects. It performs the following steps:</p>
```

Comment:
this should be updated to describe the third argument

Reply:
Done